WordPress/gutenberg
# Fix/remove term sidebar from UI side

## What?

Replacing term "sidebar" with "panel" from docs and user-facing side.

## Why?


Closes #61499

Reasoning on the linked issue. 

## How?

In the linked issue there listed cases where term "sidebar" has been used. Found those places and replaces the term with "panel"

## Testing Instructions





Read the diff and see whether terminology makes sense.

Also advisable to check WordPress site to see texts in the context of another content

### Testing Instructions for Keyboard


## Screenshots or screencast
